### PR TITLE
Dont initialize children in treeitems if not required

### PR DESCRIPTION
**Pull request by phlax**

### CODE REVIEW

File: pootle/core/mixins/treeitem.py [hunk 1]
```diff
@@ -221,7 +221,8 @@ def get_critical_url(self, **kwargs):
 
     def get_stats(self, include_children=True):
         """get stats for self and - optionally - for children"""
-        self.initialize_children()
+        if include_children:
```
Comment (ta2-1) [inline pootle/core/mixins/treeitem.py]: @phlax, children are necessary here in any case. This is un-cached TreeItem and any `._calc()` method requires children.

Comment (phlax) [inline pootle/core/mixins/treeitem.py]: @ta2-1 - ok ill remove this line

Comment (phlax) [inline pootle/core/mixins/treeitem.py]: there was a mixup  - this line is good  - other line removed

**Comment by ta2-1** [comment]
gtm

**Pull request merged**